Makefile

```diff
@@ -439,6 +436,18 @@ else
 	scripts/push-to-kind.sh -i ${VLOGGER_IMG}
 endif
 
+.PHONY: docker-push-vproxy
+docker-push-vproxy:  ## Push vertica logger docker image
+ifneq ($(strip $(VPROXY_IMG)),)
+ifeq ($(shell $(KIND_CHECK)), 0)
+	docker push ${VPROXY_IMG}
```

Comment:
should we not push the client proxy image to docker from here?

Reply:
If VPROXY_IMG is specified, we need to push it to docker.